src/main/java/view_post/use_case/ViewPostResponseModel.java

```diff
@@ -19,12 +25,26 @@ public class ViewPostResponseModel {
      * @param deadline      The string representation of the deadline of this post.
      */
     public ViewPostResponseModel(String posterEmail, String postBody, String[] postTags, String[] postReplies,
-                                 String deadline) {
+                                 String deadline, String creationDate, String collaborators, int postID, String title) {
         this.posterEmail = posterEmail;
         this.postBody = postBody;
-        this.postTags = postTags;
-        this.postReplies = postReplies;
+        String tags = "";
```

Comment:
Rather than doing all this, since the parameter `postTags` is an array of strings, you can just use `String.join`. Less likely to be buggy.